### ISSUE TITLE
Table Column - Filter - Add filter button is display even if FilterMultiple is set to false

### ISSUE THREAD

**Issue by tale80** [ant-design-blazor/ant-design-blazor#2056]
### Describe the bug

On a table column with FilterMultiple set to false, the add filter button is display.

### Steps to reproduce (please include code)
Sample column config
`<Column TData="string" Title="Market"
            DataIndex="Market"
            Sortable="false"
            Filterable
            FilterMultiple="false"
            ShowSorterTooltip="false">`

Result :
![image](https://user-images.githubusercontent.com/40459139/138459883-1118dceb-ee85-43ae-a4b2-35e454004868.png)

### Further technical details
- AntDesign Nuget Package version : 0.10.1

**Comment by ElderJames** [comment]
FilterMultiple is to enable filter for multiple colums, instead of hiding the plus button.

**Comment by alchiweb** [comment]
> FilterMultiple is to enable filter for multiple colums, instead of hiding the plus button.

Yes, this is true: `FilterMultiple` is to be use when `_columnFilterType == TableFilterType.List`.

But it could also be used when `_columnFilterType == TableFilterType.FieldType`
-> in this case, it is possible to change the line 349 in the Column.razor, with:
`@if (_columnFilterType == TableFilterType.FieldType && FilterMultiple)`

Instead, a workaround could be to use CSS to hide the "+" button...
(My use case: a backend without multiple filters)

**Comment by ElderJames** [comment]
@alchiweb Good idea, could you please create a PR?
